mass_mailing_switzerland/models/mass_mailing.py

```diff
@@ -163,7 +182,15 @@ class Mail(models.Model):
 
     @job(default_channel='root.mass_mailing')
     @related_action(action='related_action_emails')
-    @api.multi
+    @api.one
     def send_sendgrid_job(self):
+        if self.mailing_id.mailing_slug:
+            """ We append a parameter c (as campaign) to all Compassion 
+            URLs """
+            regex = r'(?<=")(https://compassion\.ch[^"]*)(?=")'
```

Comment:
What if link start with `https://www.compassion.ch` ? Don't you think we could just do for all `https://` or `http://` links?